docs/source/migration-two-dot.md

```diff
@@ -221,6 +221,42 @@ server.listen().then(({ url }) => {
 });
 ```
 
+<h3 id="existing-schema">Using Schema as Objects</h3>
+
+Some GraphQL server implementations use a GraphQL schema defined as nested objects. Apollo Server 2 accepts these schemas directly to the `ApolloServer` constructor. The following snippet demonstrates how the same schema as above might have been defined and can be passed to Apollo Server.
+
+```js
+const {
+  graphql,
+  GraphQLSchema,
+  GraphQLObjectType,
+  GraphQLString
+} = require('graphql');
+const { ApolloServer } = require('apollo-server');
+
+const schema = new GraphQLSchema({
+  query: new GraphQLObjectType({
+    name: 'RootQueryType',
+    fields: {
+      hello: {
+        type: GraphQLString,
+        resolve() {
+          return 'hello world';
+        }
+      }
+    }
+  })
+});
+
+const server = new ApolloServer({ schema });
+
+server.listen().then(({ url }) => {
+  console.log(`🚀 Server ready at ${url}`);
+});
+```
+
+> Note: While this pattern is possible, for new implementations we recommend defining `typeDefs` and `resolvers` or using `makeExecutableSchema` if necessary.
```

Comment:
I think my suggestion above might cover this disclaimer and allow it to be removed.